base/binaryplatforms.jl

```diff
@@ -7,6 +7,60 @@ export AbstractPlatform, Platform, HostPlatform, platform_dlext, tags, arch, os,
        select_platform, platforms_match, platform_name
 import .Libc.Libdl
 
+### Submodule with information about CPU features
+module CPUID
+
+export cpu_isa
+
+"""
+    ISA(features::Set{String})
+
+A structure which represents the Instruction Set Architecture (ISA) of a
+computer.  It holds the `Set` of features of the CPU.
```

Comment:
Can you document where the features are stored?  E.g. `These features are autogenerated from the `features_*.h` headers in the Julia C source code...` or something like that?